data/market-news.js

```diff
@@ -0,0 +1,44 @@
+const {stockNewsList} = require("../dal/stock-markets");
+const MARKET_NEWS = [{
+    country: 'USA',
+    news_sources: [
+        {
+            id: "Bloomberg",
```

Comment:
inconsistency: should be called "bloomberg" in lower case. make sure to change in mongoDB too